packages/terra-form-radio/tests/jest/RadioField.test.jsx

```diff
@@ -1,70 +1,119 @@
 import React from 'react';
 import ThemeContextProvider from 'terra-theme-context/lib/ThemeContextProvider';
+import uniqueid from 'lodash.uniqueid';
 
 /* eslint-disable-next-line import/no-extraneous-dependencies */
 import { shallowWithIntl, mountWithIntl } from 'terra-enzyme-intl';
 import RadioField from '../../src/RadioField';
 import Radio from '../../src/Radio';
 
+jest.mock('lodash.uniqueid');
 window.matchMedia = () => ({ matches: true });
 
+let userAgentGetter;
+beforeEach(() => {
+  userAgentGetter = jest.spyOn(window.navigator, 'userAgent', 'get');
+  uniqueid.mockImplementation(() => 'uuid123');
+});
+
 it('should render a default radio field', () => {
-  const radioField = (<RadioField legend="Default RadioField" />);
+  const radioField = <RadioField legend="Default RadioField" />;
   const wrapper = shallowWithIntl(radioField);
   expect(wrapper).toMatchSnapshot();
 });
 
+it('should render radio field with div element for Safari browser or Edg browser', () => {
+  userAgentGetter.mockReturnValue('Safari Edg');
+  const wrapper = shallowWithIntl(<RadioField legend="Custom Message RadioField" />);
+  expect(wrapper.dive()).toMatchSnapshot();
+});
```

Comment:
Can we add explicit testing for the div element? similar to:
 https://github.com/cerner/terra-core/blob/0c7a90d354e215273f20d9a8841e82a44c3cef9c/packages/terra-alert/tests/jest/Alert.test.jsx#L23-L26

Reply:
i have made changes as per suggestions.